ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-tech-preview.yaml

```diff
@@ -0,0 +1,108 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  elasticsearch-operator-src:
+    name: "4.7"
+    namespace: ocp
+    tag: elasticsearch-operator-src
+  ocp_4.6_cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  ocp_4.7_cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+  openshift_release_golang-1.14:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- from: base
+  inputs:
+    ocp_4.7_cli:
+      as:
+      - registry.svc.ci.openshift.org/ocp/4.7:cli
+    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
+  to: cluster-logging-operator
+- dockerfile_path: olm_deploy/operatorregistry/Dockerfile
+  from: base
+  to: cluster-logging-operator-registry
+- dockerfile_path: Dockerfile.e2e
+  from: elasticsearch-operator-src
+  to: cluster-logging-operator-e2e
+promotion:
+  name: tech-preview
+  namespace: logging
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
```

Comment:
this tag specification(here and in your other configs) means that PRs against each repo will not pick up the TP images from other repos, because their base/stable set of images are all going to come from the ocp:4.7 imagestream, not your logging:tech-preview imagestream.  That seems incorrect.

That said, i do not think you can just use "logging:tech-preview" as your tag_specification since it won't contain a full set of OCP images.  This may be something you need to talk to DPTP about in terms of the best approach to combine "images built by this PR + tech-preview images from other logging repos + master images from all other ocp repos"